app/javascript/src/components/Dashboard/Notes/NoteCard.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+
+import { MenuVertical, Clock } from "neetoicons";
+import { Typography, Dropdown, Tag, Avatar, Tooltip } from "neetoui";
+
+import { calculateCreatedAgo, dayTimeFormatter } from "utils/date";
+
+import { SUB_TEXT_COLOR, PROFILE_PICTURE_URL } from "./constants";
+
+const NoteCard = ({ title, description, created_at: createdAt }) => {
+  const createdAgo = calculateCreatedAgo(createdAt);
+  const dayTimeFormattedDate = dayTimeFormatter(createdAt);
```

Comment:
These variables are unnecessary. Directly invoke the util function wherever value is required. If the same value needs to be reused then only it makes sense to have these variables.